flask_restalchemy/resources/utils.py

```diff
@@ -86,13 +86,20 @@ def query_from_request(model, model_serializer, request):
     :param class model:
         SQLAlchemy model class representing a database resource
 
+    :param model_serializer:
+        instance of model serializer
+
     :param request:
         Flask http request data
 
-    :rtype: list
-    :return: the items for the current page
+    :param query:
+        SQLAlchemy query instance
+
+    :rtype: list or dict (pagination)
+    :return: the serialized response with the list or the page of items
```

Comment:
Tell more about the response format: "if 'page' is defined in the query params, a dict with blah blah blah is returned, otherwise returns a list of serialized objects"

Reply:
OK